__tests__/get-articles-api.test.js

```diff
@@ -38,4 +38,23 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("responds with the article of the given query", () => {

```

Comment:
What about including tests for a topic with no articles and a topic that doesn't exist?